components/style/properties/properties.mako.rs

```diff
@@ -2002,15 +2030,16 @@ pub mod style_structs {
                     self.text_decoration_line.contains(longhands::text_decoration_line::SpecifiedValue::LINE_THROUGH)
                 }
             % elif style_struct.name == "Box":
-                /// Sets the display property, but without touching
-                /// __servo_display_for_hypothetical_box, except when the
-                /// adjustment comes from root or item display fixups.
-                pub fn set_adjusted_display(&mut self,
-                                            dpy: longhands::display::computed_value::T,
-                                            is_item_or_root: bool) {
+                /// Sets the display property, but without touching original_display,
+                /// except when the adjustment comes from root or item display fixups.
+                pub fn set_adjusted_display(
+                    &mut self,
+                    dpy: longhands::display::computed_value::T,
+                    is_item_or_root: bool
+                ) {
                     self.set_display(dpy);
```

Comment:
Oh, the failures are because this is calling `set_display`, which also sets `original_display`. This should just do: `self.display = dpy;` instead of calling that.

Sorry for missing that!

Reply:
Thanks for the checking!
I was also thinking about this line is the root cause yesterday.
I will update the PR later!